classes/controllers/FrmWelcomeTourController.php

```
@@ -0,0 +1,588 @@
+			}
+
+			self::$steps['steps'][ $index ]['completed'] = $completed_step;
+		}//end foreach
+
+		self::$checklist['active_step'] = $active_step;
+
+		if ( $active_step === count( $step_keys ) ) {
+			self::$checklist['done']            = true;
+			self::$checklist['active_step_key'] = 'completed';
+
+			foreach ( self::$checklist['completed_steps'] as $step_key => $completed ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$completed'. (undefined)

(UnusedLocalVariable)